src/i18n.js

```diff
@@ -19,7 +19,7 @@ i18n
       escapeValue: false // not needed for react as it escapes by default
     },
     backend: {
-      loadPath: '/locales/{{lng}}.json'
+      loadPath: './locales/{{lng}}.json'
```

Comment:
This was required to build the ipfs version (was used for testing) and should be reverted